openml/testing.py

```diff
@@ -28,6 +28,9 @@ class TestBase(unittest.TestCase):
     Currently hard-codes a read-write key.
     Hopefully soon allows using a test server, not the production server.
     """
+    tracker = {}  # type: dict
+    test_server = None
```

Comment:
Since this is hardcoded below, you could move the hardcoded variable assignment here and remove the line below?

Reply:
sure, will do so. didn't want to tinker too much with existing code.